app/components/create-vhost/component.js

```diff
@@ -0,0 +1,40 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  model: null,
+  services: null,
+  certificates: null,
+  vhostService: null,
+  hasDefaultVhost: false,
+
+  setInitialVhostServiceState: function() {
+    this.setHasDefaultVhost(this.get('vhostService'));
+  }.on('didInsertElement'),
+
+  vhostServiceObserver: function() {
+    this.setHasDefaultVhost(this.get('vhostService'));
+  }.observes('vhostService'),
+
+  setHasDefaultVhost: function(vhostService) {
+    if(vhostService) {
+      vhostService.get('vhosts').then((vhosts) => {
+        let filtered = vhosts.filter((vhost) => vhost.get('isDefault'));
+        this.set('hasDefaultVhost', filtered.length > 0);
+      });
+    }
+  },
```

Comment:
Hmm, that's doesn't work for me sadly.. the state is always false with this, even for services where I know a default vhost exists.